### ISSUE TITLE
Update README for foundation.mozilla.org

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#1447]
### Issue
Now that we have a Wagtail based page (moving away from Mezzanine), we should update the "Read Me" document so people can reproduce the site locally when needed.

I saw the "Read Me" has been updated but I am not positive it is 100% updated yet - I connected with @gvn on that and it seems some parts need to be updated.

@alanmoo assigning you for triage :)

**Comment by alanmoo** [comment]
Do you mean if they're spinning up the site locally and want to populate the database so they can have a local approximation of the live site? Chris is working on that as part of #1308

**Comment by taisdesouzalessa** [comment]
I mean that (populate the database) and any step by step instructions that may have changed with the Wagtail implementation on how to build the site locally.

**Comment by alanmoo** [comment]
Yeah, this is already happened/in progress as other work happens so I'm going to close this issue.
